ipatests/test_integration/test_caless.py

```diff
@@ -306,21 +303,53 @@ def get_replica_filename(self, replica):
                             'replica-info.gpg')
 
     @classmethod
-    def export_pkcs12(cls, nickname, filename='server.p12', password=None):
-        """Export a cert as PKCS#12 to the given file"""
+    def create_pkcs12(cls, nickname, filename='server.p12', password=None):
+        """Create a cert chain and generate pkcs12 cert"""
         if password is None:
             password = cls.cert_password
-        ipautil.run(['pk12util',
-                     '-o', filename,
-                     '-n', nickname,
-                     '-d', 'nssdb',
-                     '-K', cls.cert_password,
-                     '-W', password], cwd=cls.cert_dir)
+
+        fname_chain = []
+
+        key_fname = os.path.join(cls.cert_dir, nickname + '.key')
+        certchain_fname = os.path.join(cls.cert_dir, nickname + '.pem')
+
+        nick_chain = nickname.split(os.sep)
```

Comment:
This is actually wrong, the nicknames are (in our case) always separated by '/'. I lead you to this, but I was wrong, sorry.